Libraries/Opc.Ua.Security.Certificates/Org.BouncyCastle/X509Utils.cs

```diff
@@ -0,0 +1,293 @@
+/* Copyright (c) 1996-2020 The OPC Foundation. All rights reserved.
+   The source code in this file is covered under a dual-license scenario:
+     - RCL: for OPC Foundation members in good-standing
+     - GPL V2: everybody else
+   RCL license terms accompanied with this source code. See http://opcfoundation.org/License/RCL/1.00/
+   GNU General Public License as published by the Free Software Foundation;
+   version 2 of the License are accompanied with this source code. See http://opcfoundation.org/License/GPLv2
+   This source code is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*/
+
+#if !NETSTANDARD2_1
+using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Org.BouncyCastle.Asn1;
+using Org.BouncyCastle.Asn1.X509;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.Crypto.Prng;
+using Org.BouncyCastle.Math;
+using Org.BouncyCastle.OpenSsl;
+using Org.BouncyCastle.Pkcs;
+using Org.BouncyCastle.Security;
+using Org.BouncyCastle.X509;
+
+namespace Opc.Ua.Security.Certificates.BouncyCastle
+{
+    /// <summary>
+    /// Secure .Net Core Random Number generator wrapper for Bounce Castle.
+    /// Creates an instance of RNGCryptoServiceProvider or an OpenSSL based version on other OS.
+    /// </summary>
+    public class CertificateFactoryRandomGenerator : IRandomGenerator, IDisposable
```

Comment:
Please use one file per class

Reply:
bah. Bouncy Castle is completely overvalued